### DISCUSSION TITLE
Bug: Underscores in Smartnaming rule locator are removed from the saved video title

### DISCUSSION THREAD

**Discussion by spooks7er** [aclap-dev/video-downloadhelper#551] [Help and Q&A]
### Your question

Having a Smartnaming rule for %title or %selector and if they contain underscores, those underscores are removed from the title producing a result like this:

a-z_video_name_23_02_22__17-22-00_ --> a-zvideoname23022217-22-00

For some reason the trailing zeroes are also truncated in the file name on save, but not shown in the preview. This has been a problem from before the new update.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/9903304/89654e56-992e-4829-86fd-91ba03268b0d)

### Addon & CoApp version and details

Version 9.0.1.18
Companion App installed (v2.0.19)
Edge
Windows 11

**Comment by sendittothenewts** [comment]
As well as the underscore, I have also observed it deleting commas, semicolons, ampersands, brackets, the ™ symbol, and replacing full stops with spaces.  This happens on Firefox/Linux, so it's not platform-specific.

As a workaround, switching to the old UI allows these characters again.
